composer.lock

```diff
@@ -589,30 +589,30 @@
         },
         {
             "name": "doctrine/instantiator",
-            "version": "1.5.0",
+            "version": "2.0.0",
             "source": {
                 "type": "git",
                 "url": "https://github.com/doctrine/instantiator.git",
-                "reference": "0a0fa9780f5d4e507415a065172d26a98d02047b"
+                "reference": "c6222283fa3f4ac679f8b9ced9a4e23f163e80d0"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/doctrine/instantiator/zipball/0a0fa9780f5d4e507415a065172d26a98d02047b",
-                "reference": "0a0fa9780f5d4e507415a065172d26a98d02047b",
+                "url": "https://api.github.com/repos/doctrine/instantiator/zipball/c6222283fa3f4ac679f8b9ced9a4e23f163e80d0",
+                "reference": "c6222283fa3f4ac679f8b9ced9a4e23f163e80d0",
                 "shasum": ""
             },
             "require": {
-                "php": "^7.1 || ^8.0"
+                "php": "^8.1"
```

Comment:
could this cause us to unintentionally drop 7.4 compat?

Reply:
I don't think so.

There aren't any required libraries that are consumed by users. So there's nothing that Composer knows about that users on 7.4 would care about. For our 7.4 testing, we `composer update` specifically _because_ some of the dependencies are different between 7.x and 8.x (for the 8.x builds we don't need to `composer update`, we can just `composer install`). I think this must've gotten auto-changed due to some dependency package requiring a higher version of PHP.